app/views/external_users/claim_types/selection.html.haml

```diff
@@ -24,5 +24,5 @@
 
   .form-section
     .form-group.form-buttons
-      = submit_tag t('.continue'), class: 'button'
-      = link_to t('.cancel'), external_users_root_path, role: 'button', class: 'link-secondary'
+      = govuk_button() { t('.continue') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)